src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -1210,3 +1210,121 @@
     text: |
         az monitor clone --source-resource /subscriptions/{subscriptionID}/resourceGroups/Space1999/providers/Microsoft.Compute/virtualMachines/vm1 --target-resource /subscriptions/{subscriptionID}/resourceGroups/Space1999/providers/Microsoft.Compute/virtualMachines/vm2
 """
+
+helps['monitor private-link-scope'] = """
+type: group
+short-summary: Manage monitor private link scope resource.
+"""
+
+helps['monitor private-link-scope create'] = """
+type: command
```

Comment:
Remember to add examples later

Reply:
user just need provide the name of private link scope to create it. I think we don't need example for this command. I have added examples for `scoped-resource create` and `private-endpoint-connection approve`